src/Symfony/Contracts/Translation/TranslatorTrait.php

```diff
@@ -41,7 +41,7 @@ public function getLocale()
     /**
      * {@inheritdoc}
      */
-    public function trans($id, array $parameters = [], $domain = null, $locale = null)
+    public function trans($id, array $parameters = [], $domain = null, $locale = null, $behaveSilently = true)
```

Comment:
That's a BC break.
Instead, we should catch the exception and deal with it when calling this method.